docs/language/functions.rst

```diff
@@ -180,6 +180,39 @@ see the section on :ref:`account_meta`.
 
 If ``{accounts}`` is not specified, then all account are passed.
 
+Passing seeds with external calls on Solana
+___________________________________________
+
+The Solana runtime allows you to specify the seeds to be passed for an
+external call. This is used for program derived addresses: the seeds are
+hashed with the called program id and the called account, to create program
+derived addresses. They automatically have the signer bit
+set, which allows a contract to run without an owner.
+
+.. code-block:: solidity
+
+    import 'solana';
+
+    contract c {
+        address constant program_id = address"mv3ekLzLbnVPNxjSKvqBpU3ZeZXPQdEC3bp5MDEBG68";
+
+        function test(address addr, bytes seed) public {
+            bytes instr = new bytes(1);
+
+            instr[0] = 1;
+
+            AccountMeta[1] metas = [
+                AccountMeta({pubkey: addr, is_writable: true, is_signer: true})
+            ];
+
+            token.call{accounts: metas, seeds: [ [ seed ] ]}(instr);
```

Comment:
I think it is worth writing a few words here or in the `Types` section to explain what a slice is and how to create one. Your example explains well how to use seeds with external calls, but I believe a few words on the expected type of the argument are really helpful.

Reply:
You're right, this section could use a lot more words. I've expanded it.